app/models/user.rb

```diff
@@ -69,6 +69,16 @@ def citizen?
     role == :citizen
   end
 
+  def owned_neighbourhoods
+    neighbourhoods.collect { |unit| unit.descendants }.flatten + neighbourhoods
+  end
+
+  def can_alter_neighbourhood?(neighbourhood)
+    return true if owned_neighbourhoods.include? neighbourhood
```

Comment:
think this whole method can just be replaced with `owned_neighbourhoods.include? neighbourhood`

Reply:
done!